File: C/1046.c
#include <stdio.h>
 
int main() {
 
    int a,b;
    scanf("%d%d", &a,&b);
    if(a>=b){
        a = a-b;
        a = 24 - a;
    }else{
        a = b - a;
    }
    printf("O JOGO DUROU %d HORA(S)\n", a);
}

File: Python/1046.py
ent = input().split()
inicio = int(ent[0])
fim = int(ent[1])

if inicio > fim:
    tempo = (24-inicio) + fim
    print(f'O JOGO DUROU {tempo} HORA(S)')
elif inicio < fim:
    tempo = fim - inicio
    print(f'O JOGO DUROU {tempo} HORA(S)')
elif inicio == fim:
    print('O JOGO DUROU 24 HORA(S)')